src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs

```diff
@@ -774,7 +774,11 @@ private PackageSpec GetPackageSpec(IMSBuildProject project, IReadOnlyDictionary<
             restoreMetadata.CacheFilePath = NoOpRestoreUtilities.GetProjectCacheFilePath(outputPath, project.FullPath);
             restoreMetadata.ConfigFilePaths = settings.GetConfigFilePaths();
             restoreMetadata.OutputPath = outputPath;
-            restoreMetadata.OriginalTargetFrameworks.AddRange(targetFrameworkInfos.Select(e => e.TargetAlias));
+            targetFrameworkInfos.ForEach(tfi =>
+                restoreMetadata.OriginalTargetFrameworks.Add(
+                        !string.IsNullOrEmpty(tfi.TargetAlias) ?
+                            tfi.TargetAlias :
+                            tfi.FrameworkName.GetShortFolderName()));
```

Comment:
Didn't Nikolche add this in a PR recently?  Is this a bad rebase conflict, or did you copy the pattern into a new place that wasn't in Nikolche's PR?

Reply:
Some git chery-picking problem. I just reverted them.